ImageFeed/Services/ImagesListService.swift

```diff
@@ -14,23 +14,31 @@ final class ImagesListService {
     func fetchPhotosNextPage() {
         assert(Thread.isMainThread)
         if task != nil { return }
-        lastLoadedPage = lastLoadedPage == nil ? 1 : lastLoadedPage! + 1
-        let request = makeRequest(token: storageToken.token!, page: lastLoadedPage!)
+        
+        if var lastLoadedPage {
+            self.lastLoadedPage! += 1
```

Comment:
оу, да, мой вариант оказался не таким уж и хорошим, так как переопредляет имя переменной, но даже так var - не нужен, и опционал тоже. 
```suggestion
        if lastLoadedPage != nil {
            lastLoadedPage += 1
```

Reply:
Без опционала не обойтись